sidebars.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
       collapsed: false,
       items: [
         'angular/overview',
+        'angular/build-options',
```

Comment:
Should we also link to this page in the "Resources" cards in the [overview](https://ionic-docs-git-4612-docs-ionic1.vercel.app/docs/angular/overview#resources)? I don't think it's necessary, but it might be nice.

Reply:
Great idea! Added in https://github.com/ionic-team/ionic-docs/pull/3115/commits/640789b333e62e40fd07c690be8e1fc1200131c4.